src/services/sessions.service.ts

```diff
@@ -74,12 +81,19 @@ export class SessionsService {
 
 			const logRecords = (response as any)?.log?.records;
 			const directRecords = response.records;
+			const outputs = response.records.map((record) => {
+				return {
+					...record,
+					outputs: record.callAttemptComplete?.result?.value?.toJson(),
+				};
+			});
 
 			const records = logRecords || directRecords || [];
 
 			return {
 				data: {
 					records,
+					outputs,
```

Comment:
In the function named `getLogRecordsBySessionId` we need to return only log records. If we want outputs - we should create another function on simpler - filter it in the component from the log records.